pkgs/by-name/sq/sqruff/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "sqruff";
+  version = "0.11.0";
+  src = fetchurl {
+    url = "https://github.com/quarylabs/sqruff/releases/download/v{finalAttrs.version}/sqruff-linux-x86_64-musl.tar.gz";
+    hash = "sha256-FTBwoOvKHyFXmnpHMdXVBbP7KxY1eSkAodAZtslzhIw=";
+  };
+
+  phases = [
+    "unpackPhase"
+    "installPhase"
+    "patchPhase"
+  ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp sqruff $out/bin/sqruff
+    chmod +x $out/bin/sqruff
+  '';
+
+  meta = with lib; {
+    description = "Fast SQL formatter/linter";
+    license = licenses.asl20;
+    mainProgram = "sqruff";
+    maintainers = with maintainers; [ cafkafk ];
+    platforms = platforms.linux;
```

According to the PR description shouldn't this be the following?
```suggestion
    platforms = ["x86_64-linux"];
```